README.md

```diff
@@ -126,8 +126,8 @@ docker run -d \
 ### Running using Docker compose
 
 ```sh
-docker compose up -d
-docker compose logs -f
+docker-compose up -d
+docker-compose logs -f
```

Comment:
`docker-compose` is an outdated way of calling docker compose.

https://docs.docker.com/compose/releases/migrate/#docker-compose-vs-docker-compose